Pipfile

```diff
@@ -4,18 +4,66 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-dash = "==2.15"
-pvlib = "==0.9.1"
-pythermalcomfort = "==2.9.1"
+pre-commit = "*"
+ansi2html = "==1.9.2"
+black = "==25.1.0"
+blinker = "==1.9.0"
+bump2version = "==1.0.1"
```

Comment:
what happened here? Why did you add all these dependencies? You should let Pipenv do its job. Please revert these changes.